src/observer.js

```diff
@@ -273,6 +248,32 @@ const reactiveMixin = {
     }
 }
 
+function makeObservableProp(target, propName) {
+    const valueHolderKey = Symbol(propName + " value holder")
+    const atomHolderKey = Symbol(propName + " atom holder")
+    function getAtom() {
+        return this[atomHolderKey] || (this[atomHolderKey] = createAtom("reactive " + propName))
```

Comment:
could we make this a non-enumerable prop?

Reply:
Yes. Will address this a bit later today.